app/src/main/java/org/schulcloud/mobile/controllers/course/CourseAdapter.kt

```diff
@@ -18,6 +19,7 @@ class CourseAdapter(private val onSelected: (String) -> Unit) :
     override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): CourseViewHolder {
         val binding = ItemCourseBinding.inflate(LayoutInflater.from(parent.context), parent, false)
         binding.onSelected = onSelected
+        binding.materialCard.setForegroundForJellyBean(binding.materialCard.context)
```

Comment:
Is there a reason to not just overwrite `MaterialCardView`? Having to manually invoke the fix every time seems error-prone.

Reply:
I've just discovered that this bug is fixed in the 1.1.0-alpha04 material version anyway. So I'll just update the dependency and remove the extra function.